examples/enclosure.tf

```diff
@@ -0,0 +1,22 @@
+provider "oneview" {
+	ov_username = "administrator"
+	ov_password = "madhav123"
+	ov_endpoint = "https://10.170.16.44"
+	ov_sslverify = false
+	ov_apiversion = 800
+	ov_ifmatch = "*"
+}
+
+resource "oneview_enclosure" "enclosure_inst" {
+	enclosure_group_uri = "/rest/enclosure-groups/22b775e2-57dd-46d1-beb1-b53b14d51ce4"
+	host_name = "172.18.1.13"
+	user_name = "dcs"
+	password = "dcs"
+	licensing_intent = "Oneview"
+	initial_scope_uris = ["/rest/scopes/bf7df2e6-ebbb-4c6b-bc7a-be8cdac930bf"]
```

Comment:
just a thought...can we pass scope name here and then use GetScopesByName to get the scope in create

Reply:
Same as priyankas and for enclosure group as well

Reply:
Will be taken as Enchancement